app/models/manageiq/providers/hawkular/inventory/parser/middleware_manager.rb

```diff
@@ -194,7 +194,7 @@ def fetch_server_availabilities(feeds)
           props = server.properties
 
           props['Availability'], props['Calculated Server State'] =
-            process_server_availability(props['Server State'], availability.try(:[], 'data').try(:first))
+            process_server_availability(props['Suspend State'] == 'SUSPENDED' ? 'suspended' : props['Server State'], availability.try(:[], 'data').try(:first))
```

Comment:
This should be on `process_server_availability`.

Reply:
The problem is that this function is used by another parser, I though about but in https://github.com/ManageIQ/manageiq-providers-hawkular/blob/c120d9b8306db304c6ea88cf26f5a86bbed0450c/app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/stream.rb#L71 is different so I need to see how to do it.